### PR TITLE
Dates

### PR DESCRIPTION
**Pull request by MarekBodingerBA**
Commit by commit

### CODE REVIEW

File: next/components/Atoms/DateCardDispaly.tsx [hunk 1]
```diff
@@ -11,14 +12,46 @@ interface DateCardProps {
...
 
-  const text = locale == 'en' ? `${month}. ${day}.` : `${day}. ${month}.`
+    return null
+  }, [formatter, dateFrom, dateTo])
 
   return (
     <div className={cx('m-auto text-center', wrapperClass)}>
-      <div className={cx(textSize)}>{text}</div>
+      <div className={cx(textSize)}>{dateToDisplay}</div>
       {/* <div className="text-sm capitalize">{month}</div> */}
```
Comment (radoslavzeman) [inline next/components/Atoms/DateCardDispaly.tsx]: I think we can start removing code commented out. :)

File: next/components/Atoms/DateCardDispaly.tsx [hunk 2]
```diff
@@ -11,14 +12,46 @@ interface DateCardProps {
 }
 
 const DateCardDisplay = ({ dateFrom, dateTo, textSize, wrapperClass }: DateCardProps) => {
```
Comment (radoslavzeman) [inline next/components/Atoms/DateCardDispaly.tsx]: There is typo in file name "dispaly" (the const is okay :) I just put the comment here)

File: next/components/AppLayout/Navigation/SubnavigationColumn.tsx [hunk 3]
```diff
@@ -50,11 +50,14 @@ const Column = ({ section, latestEvents, classNames }: ColumnProps) => {
...
-                        <NavigationMenu.Link className="h-10 pt-4 text-foreground-body" tabIndex={-1}>
+                        <NavigationMenu.Link
+                          className="h-10 pt-4 text-foreground-body"
+                          tabIndex={-1}
+                        >
                           <Link href={`/${event.attributes?.slug}`} passHref>
                             <a href={`/${event.attributes?.slug}`} className="flex">
                               <div className="flex h-16 w-20 bg-promo-yellow text-center">
-                                <DateCardDisplay
+                                <EventDetailsDateBox
```
Comment (radoslavzeman) [inline next/components/AppLayout/Navigation/SubnavigationColumn.tsx:58]: The text color of date should be darker as everywhere else.
image

**Review by radoslavzeman — APPROVED** [review]

File: next/components/Atoms/DateCardDispaly.tsx (hunk 2 above)
Comment (MarekBodingerBA) [reply next/components/Atoms/DateCardDispaly.tsx]: Fixed in next commit.

File: next/components/AppLayout/Navigation/SubnavigationColumn.tsx (hunk 3 above)
Comment (MarekBodingerBA) [reply next/components/AppLayout/Navigation/SubnavigationColumn.tsx]: Fixed.

**Pull request merged**